src/@ionic-native/plugins/clover-go/index.ts

```diff
@@ -0,0 +1,90 @@
+import { Injectable } from '@angular/core';
+import { Plugin, Cordova, CordovaProperty, CordovaInstance, InstanceProperty, IonicNativePlugin } from '@ionic-native/core';
+import { Observable } from 'rxjs';
+
+/**
+ * @name Clover Go
+ * @description
+ * This plugin connect to Clover Go payment device and process payments.
+ *
+ * @usage
+ * ```typescript
+ * import { CloverGo } from '@ionic-native/clover-go';
```

Comment:
Change to: `* import { CloverGo } from '@ionic-native/clover-go/ngx';`

Reply:
Updated